examples/server-side-rendering/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "importHelpers": true,
```

Comment:
This is a new requirement?

Reply:
This one is another result of the migration.
In package.json you'll also notice a new dependency [tslib].(https://www.npmjs.com/package/tslib) introduced by the migrator (I think it's optional)
This compile option is a usage requirement of this lib.

Reply:
Yep it look like it's the same purpose than Babel runtime.